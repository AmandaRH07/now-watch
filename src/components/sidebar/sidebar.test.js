import Sidebar from "."

describe('<Sidebar/>', () => {

  it('Should render Sidebar', () => {
    expect(<Sidebar  />).toMatchSnapshot()
  })
})