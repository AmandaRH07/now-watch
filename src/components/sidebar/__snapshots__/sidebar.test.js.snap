// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Sidebar/> 1`] = `<Sidebar />`;

exports[`<Sidebar/> Should call handleSaveFilter 1`] = `<Sidebar />`;

exports[`<Sidebar/> Should render  1`] = `<Sidebar />`;

exports[`<Sidebar/> Should render Sidebar 1`] = `<Sidebar />`;

exports[`<Sidebar/> Should render Sidebar 2`] = `<Sidebar />`;

exports[`<Teste/> Deve renderizar o card sem problemas 1`] = `<Sidebar />`;

exports[`<Teste/> Deve renderizar o card sem problemas 2`] = `<Sidebar />`;
